x/inflation/v1/keeper/integration_test.go

```diff
@@ -147,113 +201,144 @@ var _ = Describe("Inflation", Ordered, func() {
+					Expect(allocatedAmt.IsZero()).ToNot(BeTrue())
+					Expect(allocatedAmt.GT(expected)).To(BeTrue())
 				})
 			})
 		})
 
 		Context("with inflation param disabled", func() {
 			BeforeEach(func() {
-				params := s.app.InflationKeeper.GetParams(s.ctx)
+				params := s.network.App.InflationKeeper.GetParams(s.network.GetContext())
 				params.EnableInflation = false
-				s.app.InflationKeeper.SetParams(s.ctx, params) //nolint:errcheck
+				err := integrationutils.UpdateInflationParams(
+					integrationutils.UpdateParamsInput{
+						Tf:      s.factory,
+						Network: s.network,
+						Pk:      s.keyring.GetPrivKey(0),
+						Params:  params,
+					},
+				)
+				Expect(err).ToNot(HaveOccurred(), "error while setting params")
 			})
 
 			Context("after the network was offline for several days/epochs", func() {
 				BeforeEach(func() {
-					s.CommitAfter(time.Minute)        // start initial epoch
-					s.CommitAfter(time.Hour * 24 * 5) // end epoch after several days
+					Expect(s.network.NextBlockAfter(time.Minute)).To(BeNil()) // Start Epoch
+					s.network.NextBlockAfter(time.Hour * 24 * 5)              // end epoch after several days
 				})
 				When("the epoch start time has not caught up with the block time", func() {
 					BeforeEach(func() {
 						// commit next 3 blocks to trigger afterEpochEnd let EpochStartTime
 						// catch up with BlockTime
-						s.CommitAfter(time.Second * 6)
-						s.CommitAfter(time.Second * 6)
-						s.CommitAfter(time.Second * 6)
+						Expect(s.network.NextBlockAfter(time.Second * 6)).To(BeNil())
+						Expect(s.network.NextBlockAfter(time.Second * 6)).To(BeNil())
+						Expect(s.network.NextBlockAfter(time.Second * 6)).To(BeNil())
 
-						epochInfo, found := s.app.EpochsKeeper.GetEpochInfo(s.ctx, epochstypes.DayEpochID)
-						s.Require().True(found)
+						epochInfo, found := s.network.App.EpochsKeeper.GetEpochInfo(s.network.GetContext(), epochstypes.DayEpochID)
+						Expect(found).To(BeTrue())
 						epochNumber = epochInfo.CurrentEpoch
 
-						skipped = s.app.InflationKeeper.GetSkippedEpochs(s.ctx)
+						skipped = s.network.App.InflationKeeper.GetSkippedEpochs(s.network.GetContext())
 
-						s.CommitAfter(time.Second * 6) // commit next block
+						// commit next block
+						Expect(s.network.NextBlockAfter(time.Second * 6)).To(BeNil())
 					})
 					It("should increase the epoch number ", func() {
-						epochInfo, _ := s.app.EpochsKeeper.GetEpochInfo(s.ctx, epochstypes.DayEpochID)
+						epochInfo, _ := s.network.App.EpochsKeeper.GetEpochInfo(s.network.GetContext(), epochstypes.DayEpochID)
 						Expect(epochInfo.CurrentEpoch).To(Equal(epochNumber + 1))
 					})
 					It("should not increase the skipped epochs number", func() {
-						skippedAfter := s.app.InflationKeeper.GetSkippedEpochs(s.ctx)
+						skippedAfter := s.network.App.InflationKeeper.GetSkippedEpochs(s.network.GetContext())
 						Expect(skippedAfter).To(Equal(skipped + 1))
```

Comment:
ditto, there are a lot of uses of the `App` instead of actual integration testing in these unfortunately.